modules/acquia_cms_site_studio/acquia_cms_site_studio.install

```diff
@@ -83,3 +83,21 @@ function set_theme_site_studio_as_default() {
     ->set('default', 'cohesion_theme')
     ->save();
 }
+
+/**
+ * Use IMCE for inserting images into Site Studio configuration.
+ */
+function acquia_cms_site_studio_update_8001() {
+  // Using media browser, creates problem if you ever want to export
+  // the configuration using the Site Studio package manager
+  // as the media entity won't be included as Site Studio
+  // does not export content entities as part of packages.
+  $config = \Drupal::service('config.factory')->getEditable('cohesion.settings');
+  $config->set('image_browser.config', [
```

Comment:
@chandan-singh7929 I think we should use condition check here: `if (!empty($config)) {`

Reply:
done